docs/kyma/docs/05-03-overrides.md

```diff
@@ -32,39 +29,63 @@ Overrides for the component are composed from two sets: **common** overrides and
 
 Kyma uses common overrides for the installation of all components. ConfigMaps and Secrets marked with the label `installer:overrides`, contain the definition. They require no additional label.
 
-Kyma uses component-specific overrides only for the installation of specific components. ConfigMaps and Secrets marked with both `installer:overrides` and `component: <name>` labels, where `<name>` is the component name, contain the definition. Component-specific overrides have precedence over Common ones in case of conflicting entries.
+Kyma uses component-specific overrides only for the installation of specific components. ConfigMaps and Secrets marked with both `installer:overrides` and `component: {component-name}` labels contain the definition. Component-specific overrides have precedence over common ones in case of conflicting entries.
 
 
-## Overrides Examples
+## Overrides examples
 
 ### Top-level charts overrides
 
 Overrides for top-level charts are straightforward. Just use the template value from the chart (without leading ".Values." prefix) as the entry key in the ConfigMap or Secret.
 
 Example:
 
-The Installer uses a `core` top-level chart that contains a template with the following value reference:
+The Installer uses a `asset-store` top-level chart that contains a template with the following value reference:
+
 ```
-memory: {{ .Values.test.acceptance.ui.requests.memory }}
+resources: {{ toYaml .Values.resources | indent 12 }}
 ```
-The chart's default value `test.acceptance.ui.requests.memory` in the `values.yaml` file resolves the template.
+
+The chart's default values `minio.resources.limits.memory` and `minio.resources.limits.cpu` in the `values.yaml` file resolve the template.
 The following fragment of `values.yaml` shows this definition:
 ```
-test:
-  acceptance:
-    ui:
-      requests:
-        memory: "1Gi"
+minio:
+  resources:
+    limits:
+      memory: "128Mi"
+      cpu: "100m"
 ```
 
-To override this value, for example to "2Gi", proceed as follows:
-- Create a ConfigMap in the `kyma-installer` Namespace, labelled with: `installer:overrides` (or reuse an existing one).
-- Add an entry `test.acceptance.ui.requests.memory: 2Gi` to the map.
+To override these values, for example to `512Mi` and `250m`, proceed as follows:
+- Create a ConfigMap in the `kyma-installer` Namespace, labelled with: `installer:overrides`. You can also use the existing ConfigMap.
```

Comment:
another place where we mention labels but this time just one

Reply:
I would also not mention `You can also use the existing ConfigMap.` as it is not straight forward and at the end I don't think this is recommended, it is always better to create new file then merge custom user file with default config yamls

Reply:
Ok, changed that all, hope it reads well now :)